frontend/src/pages/answer/resultsTable/SupportingPublications.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useReducer, useEffect } from 'react';
+import Paper from '@material-ui/core/Paper';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Collapse from '@material-ui/core/Collapse';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import shortid from 'shortid';
+
+function reducer(state, action) {
```

Comment:
It might be nice to name this something more informative. I'm not sure how broad the scope of this thing is, though. "selectionReducer"?

Reply:
Updated in b05f55ac3910ed20a5caa0ba0c5745bdcbe4f879

Reply:
Is there anything about this that is specific to publications? The name should describe what it does, as generally as possible. If it's used in the context of publications, so be it.

Reply:
Changed in aeb9adac67c32ad6ef20d4e54e233dc2976d88f7